__tests__/app.test.js

```diff
@@ -273,3 +273,47 @@ describe("9. GET /api/users", () => {
       });
   });
 });
+
+describe("10. GET /api/articles (queries)", () => {
+  test("Returns An article response object should also now include", () => {
+    const topic = "mitch";
+    const sort_by = ["ASC", "DECS"];
+    return request(app)
+      .get(`/api/articles/${topic}/${sort_by[1]}`)
+      .expect(200)
+      .then((data) => {
+        expect(data.body.length).toBe(11);
+      });
+  });
```

Comment:
we're not testing if it has been sorted in descending order here - which would technically fail as in your sort_by array DESC is spelt DECS, so in a way this is a false positive. 
Have a look at jest-sorted to be testing for whether the topics coming back or in the correct order

Reply:
you've still not done this one either